### PR TITLE
Pop up window and card element to Array

### PR DESCRIPTION
**Pull request by Yacoubou-seidou**
### Pop up window and card element to Array
- Add array with all projects details in javascript
- Store all minimum required data for a project in the array
- Add a modal with the correct project detail
- Close the modal when clicking on the close button
- Project section is created dynamically using the information stored in that JavaScript
- Add a modal pop-up when checking project details on mobile
- Add a modal pop-up when checking project details on desktop
- Template 1 design

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,95 +65,52 @@ <h5 tabindex="0">
...
-            <h2 tabindex="0">Multi-Post<br>
-              Stories</h2>
+  </section>
+  <div class="bottom_section">
+      <section class="about_section" tabindex="0" id="about">
+      <!-- <div id="myModal" class="modal">
+        <div class="modal-content">
+                <span class="closeModal" id="closeModal">&times;</span>
+          <div class="text_side modal_head" tabindex="0">
+            <h2 tabindex="0">Tonic</h2>
```
Comment (Ghiftee) [inline index.html]: - [ ] Kindly remove the commented code from your project has it helps you have a cleaner codebase

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @Yacoubou-seidou  and @Ferizad 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs as expected
✔️ Linters are set up correctly
✔️ README is professional

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._
- [ ] As your design should look exactly like the figma design, kindly correct the spacings in the images below:

### Your design
Screenshot 2023-03-07 at 16 28 24 (2)

### Figma design
Screenshot 2023-03-07 at 16 28 18 (2)

### Your design
Screenshot 2023-03-07 at 16 34 12 (2)

### Figma design
Screenshot 2023-03-07 at 16 29 15 (2)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: style.css [hunk 2]
```diff
@@ -27,6 +27,10 @@ label {
     width: 100%;
   }
 
+  .hide_y_scroll {
+    overflow-y: hidden;
+  }
+
```
Comment (tufoinnkuo10) [inline style.css:33]: - [optional] kinldy make the button details hover on line 304 to have the color `#6070FF`.

File: script.js [hunk 3]
```diff
@@ -3,6 +3,63 @@ const menuItems = document.querySelectorAll('.link_item');
...
+    textClass: 'second_reverse',
+  },
+  {
+    name: 'Facebook 360',
+    skills: { campany: 'FACEBOOK', stack: 'Full Stack Dev', year: '2015' },
+    description:
+      "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    technologies: ['html', 'Ruby on rails', 'css', 'javaScript'],
+    image: './pictures/SnapshootPortfolio2.png',
+    livelink: 'https://yacoubou-seidou.github.io/Portfolio/',
```
Comment (tufoinnkuo10) [inline script.js:43]: - [ ] please, like the previous reviewer noted, kindly ensure your` see live` and `seesource` buttons follow the Figma design. Please reduce the gap between them for both the mobile and desktop version
- [optional] you can as well reduce your mobile popup height, so everything can be seen without scrolling 
Yours            
Screenshot (872)_LI  
figma
  Screenshot (873)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @team

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design well followed

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @Yacoubou-seidou   👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have inspected your project and the design of the buttons looks good enough. Students are required to replicate the Figma designs but it is not always a strict requirement that it looks exactly as the Figma design looks it only has to be similar adhering to the Fonts, colors, images and typography that is provided. We understand that learning to style your projects is a process and you have done well and keep improving on your CSS skills. :fire: 

As this was the only reason that your project was not approved please go ahead and request another  review in your dashboard and the next reviewer will approve it immediately :rocket: 


In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Review by binhussen — APPROVED** [review]
Hi @Yacoubou-seidou , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**